src/cmd/src/datanode.rs

```diff
@@ -102,12 +103,13 @@ struct StartCommand {
 }
 
 impl StartCommand {
-    fn load_options(&self, top_level_opts: TopLevelOptions) -> Result<Options> {
-        let mut opts: DatanodeOptions = if let Some(path) = &self.config_file {
-            toml_loader::from_file!(path)?
-        } else {
-            DatanodeOptions::default()
-        };
+    fn load_options(
+        &self,
+        top_level_opts: TopLevelOptions,
+        env_vars_prefix: &str,
```

Comment:
It's for unit testing purposes. Because the execution of unit tests is parallel, it will get errors since the environment variables affect other test cases, for example: [test_read_from_config_file()](https://github.com/zyy17/greptimedb/blob/f8f6542b665ae4ddd11493101e06cd2cd7cf998d/src/cmd/src/frontend.rs#L296) will be affected by [test_config_precedence_order](https://github.com/zyy17/greptimedb/blob/f8f6542b665ae4ddd11493101e06cd2cd7cf998d/src/cmd/src/frontend.rs#L414). For resolve this problem, I will use the different prefix like `FRONTEND_UT`.

My initial implementation doesn't add `env_vars_prefix` argument then the unit tests always failed by the concurrent problem.

Let me think about if there is any better solutions.

Reply:
@MichaelScofield I move the `env_vars_prefix` to `StartCommnd` to make it configurable, and it looks more reasonable.